fritz

```diff
@@ -57,7 +47,7 @@ def generate_skyportal_token():
          """from skyportal.models import init_db; """
          """from skyportal.model_util import create_token; """
          """conn = init_db(**load_config(['config.yaml'])['database']); """
-         """print(create_token(['Upload data', 'Manage sources'], 1, 'kowalski'))" """
+         """print(create_token(['Upload data', 'Manage sources', 'Manage groups'], 1, 'kowalski'))" """
```

Comment:
@stefanv, can your new token stuff be used to deprecate this?

Reply:
I would hope so, but it needs to be merged first -- aren't there any clashes with Josh's stuff (Classification #2)?